build/common-replacements.js

```diff
@@ -26,3 +26,16 @@ module.exports.altIndexOfUseReplacement = [
     /(\W)([\w\.\(\),\[\]]+)(\.indexOf\()/gm
   , '$1indexOf($2, '
 ]
+module.exports.objectKeysDefine = [
```

Comment:
FWIW, in theory buffer works back to IE6. Tests used to pass but they
became flakey a while ago and I thought it was SauceLab's fault so I just
disabled IE6 and IE7 instead of debugging it. Tests still pass down to IE8.

The browserify http module claims to work down to IE5.5!
On Sun, Jun 14, 2015 at 4:37 PM Daniel Cousens <notifications@github.com>
wrote:

> In build/common-replacements.js
> <https://github.com/nodejs/readable-stream/pull/139#discussion_r32386021>:
>
> > @@ -26,3 +26,16 @@ module.exports.altIndexOfUseReplacement = [
> >      /(\W)([\w\.\(\),\[\]]+)(\.indexOf\()/gm
> >    , '$1indexOf($2, '
> >  ]
> > +module.exports.objectKeysDefine = [
>
> Any of these browsers
> <https://cloud.githubusercontent.com/assets/1128607/8131283/3a814a02-10e7-11e5-85c0-8d3887303f70.png>
> ?
>
> As far as I could tell on https://kangax.github.io/compat-table/es5/, all
> the above browsers supported Object.keys, only if we go as far back as IE8,
> FF 3.5 or Safari 4, does that actually become an issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/readable-stream/pull/139/files#r32386021>.
>